test/src/xquery/fn.xql

```diff
@@ -57,6 +57,20 @@ function fnt:doc-available($filename as xs:string) {
 };
 
 declare
+    %test:args("test.xml")
+    %test:assertEquals("true")
+function fnt:doc($filename as xs:string) {
+    fn:doc("/db/fn-test/" || $filename) instance of document-node()
+};
+
+declare
+    %test:args("test.xml")
+    %test:assertEmpty
+function fnt:doc($filename as xs:string) {
```

Comment:
@joewiz We seem to have two functions of the same name and arity, i.e. `fnt:doc#1`. This should cause the test suite to fail, although it seems to have passed on Travis at least?!?

Reply:
Drat, I had even given them unique names when testing in eXide but forgot to put this change into my comit. How would you recommend I fix this: push the fix to my branch and submit a new PR? I'd name these `fnt:doc-returns-document-node` and `fnt:doc-does-not-return-element`.

Reply:
A new PR would be great. I don't understand why the test suite doesn't currently fail though, are your tests being executed I wondered?

Reply:
@joewiz Any update on this please?

Reply:
@adamretter Thanks for the reminder. @duncdrum's comment on #944 reminded me too. A PR is coming shortly.